api/api.go

```diff
@@ -14,12 +14,29 @@ const (
 	apiTimeout = 5 * time.Second
 )
 
+// HttpGET is a utility function for making http get requests with Sia-Agent set
```

Comment:
I considered `api.Get/Post` but to me those were quite similar to `apiPost` and `apiGet` within siac and a few other locations so I chose `api.HttpGET/POST` to ensure the distinction was clear